build/nuget/XamlStyler.Console.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
 	<metadata>
 		<id>XamlStyler.Console</id>
-		<version>3.0.0.0</version>
+		<version>4.0.0.0</version>
```

Comment:
While this is certainly a major change for the console app, we'd like to keep versions in lockstep with the extension to make it easier for compatibility checks and debugging. Can we snap to 3.3.0.0? I'd like to eventually align these via build scripts in pipelines.